docs/developer-guide/extensions.md

```diff
@@ -51,7 +51,9 @@ Note that many of these extensions are no longer available in WebGL2 as the func
 | `OES_texture_float_linear`  | Enables linear filter for float textures | |
 | `OES_fbo_render_mipmap`     | Render to specific texture mipmap level | |
 | `EXT_sRGB`                  | sRGB encoded rendering | |
+| `WEBGL_color_buffer_float` | framebuffer render to 32 bit float color buffer | |
```

Comment:
Move this back to WebGL 2. It's written against the WebGL 2 spec (see dependencies: https://www.khronos.org/registry/webgl/extensions/EXT_color_buffer_float/).

WebGL 1 implementations implicitly enable it via OES_texture_float: https://www.khronos.org/registry/webgl/extensions/OES_texture_float/

Reply:
[WEBGL_color_buffer_float](https://www.khronos.org/registry/webgl/extensions/WEBGL_color_buffer_float/) is WebGL1 only, you are referring to [EXT_color_buffer_float](https://www.khronos.org/registry/webgl/extensions/EXT_color_buffer_float/) which is WebGL2 only and listed below.